mlflow/models/utils.py

```diff
@@ -146,10 +156,24 @@ def _read_example(mlflow_model: Model, path: str):
     if mlflow_model.saved_input_example_info is None:
         return None
     example_type = mlflow_model.saved_input_example_info["type"]
-    if example_type != "dataframe":
+    if example_type not in ["dataframe", "ndarray"]:
         raise MlflowException(
             "This version of mlflow can not load example of type {}".format(example_type)
         )
     input_schema = mlflow_model.signature.inputs if mlflow_model.signature is not None else None
     path = os.path.join(path, mlflow_model.saved_input_example_info["artifact_path"])
-    return _dataframe_from_json(path, schema=input_schema, precise_float=True)
+    if (example_type == "ndarray") or (input_schema and input_schema.is_tensor_spec()):
```

Comment:
I think the example type must be "ndarray", otherwhise we can not read it with `_read_tensor_input_from_json`